src/glb-healthcheck/main.go

```diff
@@ -77,15 +83,42 @@ Options:
 		return
 	}
 
-	// the check manager will run the HC loop and manage most of the HC part of the work
-	ctx.checkManager = NewHealthCheckManager(HealthCheckTimeout, HealthCheckInterval)
-
 	// load up the forwarding table and register all the check targets
 	err = ctx.LoadForwardingTable()
 	if err != nil {
 		ctx.logContext.Fatalf("Could not load initial forwarding table: %v", err)
 		return
 	}
+	
+	HealthCheckTimeout := DefaultHealthCheckTimeout
+	if ctx.forwardingTableConfig.HealthcheckGlobalCfg != nil &&
+		ctx.forwardingTableConfig.HealthcheckGlobalCfg.TimeoutMilliSec != 0 {
+		HealthCheckTimeout = ctx.forwardingTableConfig.HealthcheckGlobalCfg.TimeoutMilliSec * time.Millisecond
+	}
+
+	HealthCheckInterval := DefaultHealthCheckInterval
+	if ctx.forwardingTableConfig.HealthcheckGlobalCfg != nil &&
+		ctx.forwardingTableConfig.HealthcheckGlobalCfg.IntervalMilliSec != 0 {
+		HealthCheckInterval = ctx.forwardingTableConfig.HealthcheckGlobalCfg.IntervalMilliSec * time.Millisecond
+	}
+
+	SuccessesBeforeMarkedHealthy := DefaultSuccessesBeforeMarkedHealthy
+	FailuresBeforeMarkedFailed := DefaultFailuresBeforeMarkedFailed
+	if ctx.forwardingTableConfig.HealthcheckGlobalCfg != nil &&
+		ctx.forwardingTableConfig.HealthcheckGlobalCfg.Trigger != 0 {
+		SuccessesBeforeMarkedHealthy = ctx.forwardingTableConfig.HealthcheckGlobalCfg.Trigger
+		FailuresBeforeMarkedFailed   = ctx.forwardingTableConfig.HealthcheckGlobalCfg.Trigger
+	}
+
+	// the check manager will run the HC loop and manage most of the HC part of the work
+	ctx.checkManager = NewHealthCheckManager(HealthCheckTimeout, HealthCheckInterval, SuccessesBeforeMarkedHealthy,
+		FailuresBeforeMarkedFailed)
```

Comment:
Since these are now locals, the locals should start with lower to match Go idioms.
```suggestion
	healthCheckTimeout := DefaultHealthCheckTimeout
	if ctx.forwardingTableConfig.HealthcheckGlobalCfg != nil &&
		ctx.forwardingTableConfig.HealthcheckGlobalCfg.TimeoutMilliSec != 0 {
		healthCheckTimeout = ctx.forwardingTableConfig.HealthcheckGlobalCfg.TimeoutMilliSec * time.Millisecond
	}

	healthCheckInterval := DefaultHealthCheckInterval
	if ctx.forwardingTableConfig.HealthcheckGlobalCfg != nil &&
		ctx.forwardingTableConfig.HealthcheckGlobalCfg.IntervalMilliSec != 0 {
		healthCheckInterval = ctx.forwardingTableConfig.HealthcheckGlobalCfg.IntervalMilliSec * time.Millisecond
	}

	successesBeforeMarkedHealthy := DefaultSuccessesBeforeMarkedHealthy
	failuresBeforeMarkedFailed := DefaultFailuresBeforeMarkedFailed
	if ctx.forwardingTableConfig.HealthcheckGlobalCfg != nil &&
		ctx.forwardingTableConfig.HealthcheckGlobalCfg.Trigger != 0 {
		successesBeforeMarkedHealthy = ctx.forwardingTableConfig.HealthcheckGlobalCfg.Trigger
		failuresBeforeMarkedFailed   = ctx.forwardingTableConfig.HealthcheckGlobalCfg.Trigger
	}

	// the check manager will run the HC loop and manage most of the HC part of the work
	ctx.checkManager = NewHealthCheckManager(healthCheckTimeout, healthCheckInterval, successesBeforeMarkedHealthy,
		failuresBeforeMarkedFailed)
```